packages/types/src/allForks/index.ts

```diff
@@ -1,2 +1,3 @@
-export * as ts from "./types";
-export * as ssz from "./sszTypes";
+export * from "./types.js";
```

Comment:
Why the new line `export * from "./types.js";`?

Reply:
So we can remove the package-level `allForks.js` and `allForks.d.ts`. This file is now exported at the path `./allForks`.